source/Missions/Services/Agents/Messages/AgentMovementDelta.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Missions.Services.Agents.Packets;
+using System;
+using TaleWorlds.Library;
+using TaleWorlds.MountAndBlade;
+
+namespace Missions.Services.Agents.Messages
+{
+    /// <summary>
+    /// Represents the delta of an agent's movement between the time it was created and the time it was last updated.
+    /// </summary>
+    internal sealed class AgentMovementDelta
```

Comment:
Make this into a `ProtoContract`. For an example see `AgentActionData`.

What's the methodology of making this class sealed?

Reply:
Also write a serialization test for this similar to `NetworkMessageSerializationTest`